### PR TITLE
Fix user admin authorization plugin adding only a single authorization with commas

### PR DESCRIPTION
**Pull request by joeybrk372**
- [x] joeferner
- [x] sfeng88
- [ ] mwizeman joeybrk372 jharwig

Testing Instructions: Attempt to add an authorization to a user through the admin tools user editor with commas. The user should get one authorization for each comma-separated value

CHANGELOG
Fixed: Attempting to add an authorization to a user that contained commas via the user admin authorization tool would prevent the user from logging in. Now, an authorization will be added for each value separated by commas.

### CODE REVIEW

File: web/plugins/admin-user-property-auth/src/main/java/org/visallo/web/plugin/adminUserTools/userPropertyAuth/UserAddAuthorization.java [hunk 1]
```diff
@@ -40,7 +41,10 @@ public JSONObject handle(
             throw new VisalloAccessDeniedException("Authorization repository does not support updating", authUser, userName);
         }
 
-        ((UpdatableAuthorizationRepository) authorizationRepository).addAuthorization(user, auth, authUser);
+        for (String authStr : auth.split(SEPARATOR)) {
```
Comment (jharwig) [inline web/plugins/admin-user-property-auth/src/main/java/org/visallo/web/plugin/adminUserTools/userPropertyAuth/UserAddAuthorization.java]: should we ignore spaces?

Comment (kunklejr) [inline web/plugins/admin-user-property-auth/src/main/java/org/visallo/web/plugin/adminUserTools/userPropertyAuth/UserAddAuthorization.java]: We should definitely ignore spaces. Should be as simple as calling `authStr.trim()` before the `addAuthorization` call?

Comment (joeferner) [inline web/plugins/admin-user-property-auth/src/main/java/org/visallo/web/plugin/adminUserTools/userPropertyAuth/UserAddAuthorization.java]: We should probably make that the responsibility of the `addAuthorization` method itself not the caller?

Comment (joeybrk372) [inline web/plugins/admin-user-property-auth/src/main/java/org/visallo/web/plugin/adminUserTools/userPropertyAuth/UserAddAuthorization.java]: updated to trim whitespace

**Comment by sfeng88** [comment]
=

**Pull request merged**